cfme/infrastructure/cluster.py

```diff
@@ -4,15 +4,15 @@
 :var page: A :py:class:`cfme.web_ui.Region` object describing common elements on the
            Cluster pages.
 """
+from functools import partial
```

Comment:
agree. I also use such approach but I also try to follow code conventions existing in this project.
fixing